tests/chainer_tests/functions_tests/activation_tests/test_elu.py

```diff
@@ -43,9 +43,13 @@ def setUp(self):
         if self.dtype == numpy.float16:
             self.check_forward_options.update({'atol': 5e-4, 'rtol': 5e-3})
             self.check_backward_options.update({'atol': 5e-4, 'rtol': 5e-3})
+            self.check_double_backward_options.update(
+                {'atol': 5e-4, 'rtol': 5e-3})
 
     def generate_inputs(self):
         x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        if self.test_name in ('test_backward', 'test_double_backward'):
+            x[(-0.01 < x) & (x < 0.01)] = 0.5
```

Comment:
Could you add a comment here or remove the line `dodge_nondifferentiable = True`?

Reply:
Ooops, forgot to push the removal of `dodge_nondifferentiable`.